alma/alma-installments-prestashop
# Add htaccess in builder zip

### Reason for change

[Linear task](https://linear.app/almapay/issue/MPP-842/fix-builder-to-add-htaccess-in-zip)

### Code changes

Add htaccess in builder zip

### How to test

Build the module , unzip it and open it with EDI to see if the .htaccess is in the folder

### Checklist for authors and reviewers




### Non applicable
